src/viewer/PdfJsViewer.jsx

```diff
@@ -109,7 +109,7 @@ export class PdfJsViewer extends Component {
       renderAllPages
     } = this.state
 
-    if (errored) return <NoViewer file={url} />
+    if (errored) return <NoViewer file={file} />
```

Comment:
Can you add it to proptypes ? ;) https://github.com/cozy/cozy-drive/blob/fab5d468047ab2ebb5c50892980e58d7d002193e/src/viewer/PdfJsViewer.jsx#L186